StatisticsInR/2.2_descriptive_statistics.Rmd

```diff
@@ -176,22 +250,29 @@ Q4: the maximum value in the set.
 
 Note that the set of data between Q1 and Q3 (which contains the middle 50% of the data) is the interquartile range (IQR).
 
-In R, quartiles may be obtained using the function `quantile()` (NB: this is not a typo, quantile refers to the intervals (0, 25, 50, 75, 100%) at which the quartiles, the values in the data, are assessed). 
+In R, quartiles may be obtained using the function `quantile()`.
```

Comment:
Maybe the first sentence should be about what quartiles are, or that sometimes it can be useful to get more information about the data than the mean and ranges.